services/openshift/openshift_option

```diff
@@ -3,3 +3,6 @@
 # Modify these options if you want to change openshift hostname
 OPENSHIFT_SUBDOMAIN="$(hostname).$(/opt/adb/openshift/get_ip_address).xip.io"
 IMAGE="docker.io/openshift/origin:latest"
+#NO_PROXY=localhost
```

Comment:
If this is some kind of tweak setting then can we have comment above it?
Otherwise can we remove these lines?